### PR TITLE
feat: add method which return current cdp

### PR DESCRIPTION
**Pull request by PavelMor25**
## Purpose
closes Devexpress/testcafe-private/issues/479

## Approach
_Describe how your changes address the issue or implement the desired functionality in as much detail as possible._

## References
_Provide a link to the existing issue(s), if any._

## Pre-Merge TODO
- [ ] Write tests for your proposed changes
- [ ] Make sure that existing tests do not fail

### CODE REVIEW

**Review by Aleksey28 — CHANGES_REQUESTED** [review]
Please, look at my comments. After that update type definitions here https://github.com/DevExpress/testcafe/tree/master/ts-defs-src

File: src/test-run/commands/type.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ export default {
     closeWindow:                         'close-window',
     getCurrentWindow:                    'get-current-window',
     getCurrentWindows:                   'get-current-windows',
+    getCurrentCDPSession:                'get-current-c-d-p-session',
```
Comment (Aleksey28) [inline src/test-run/commands/type.js:44]:
```suggestion
    getCurrentCDPSession:                'get-current-cdp-session',
```

File: ts-defs-src/test-api/test-controller.d.ts [hunk 2]
```diff
@@ -413,6 +413,11 @@ interface TestController {
      * Retrieves a `window` object that corresponds to the currently open window.
      */
     getCurrentWindow(): WindowDescriptorPromise;
+    
+    /**
+     * Retrieves a `Chrome DevTools Protocol` object that corresponds to the currently open window in native automation.
+     */
+    getCurrentCDPSession(): TestControllerPromise;
```
Comment (Aleksey28) [inline ts-defs-src/test-api/test-controller.d.ts:420]: It should return CDP session, not Test Controller

**Review by Aleksey28 — COMMENTED** [review]
Please, look at my comments, and also add definition tests to this directory: https://github.com/DevExpress/testcafe/blob/master/test/server/compiler-test.js
As example, you can explore how it was made for `getCurrentWindow`.

File: ts-defs-src/test-api/test-controller.d.ts [hunk 3]
```diff
@@ -413,6 +415,11 @@ interface TestController {
      * Retrieves a `window` object that corresponds to the currently open window.
      */
     getCurrentWindow(): WindowDescriptorPromise;
+    
+    /**
+     * Retrieves a `Chrome DevTools Protocol` object that corresponds to the currently open window in native automation.
+     */
+    getCurrentCDPSession(): CDPSessionPromise;
```
Comment (PavelMor25) [inline ts-defs-src/test-api/test-controller.d.ts:422]: @titerman, please check the description.

**Review by Aleksey28 — APPROVED** [review]

File: ts-defs-src/test-api/test-controller.d.ts [hunk 4]
```diff
@@ -413,6 +415,11 @@ interface TestController {
      * Retrieves a `window` object that corresponds to the currently open window.
      */
     getCurrentWindow(): WindowDescriptorPromise;
+    
+    /**
+     * Retrieves a `Chrome DevTools Protocol` object that corresponds to the currently open window in native automation.
```
Comment (titerman) [inline ts-defs-src/test-api/test-controller.d.ts:420]:
```suggestion
     * Retrieves a `Chrome DevTools Protocol` object that corresponds to the currently open window (native automation only).
```

File: ts-defs-src/test-api/test-controller.d.ts (hunk 3 above)
Comment (titerman) [reply ts-defs-src/test-api/test-controller.d.ts:422]: One minor edit 👍

**Review by titerman — APPROVED** [review]

**Pull request merged**